### PR TITLE
Task #6144 - Login with google

### PR DESCRIPTION
**Pull request by taylq**
https://edu-redmine.framgia.vn/issues/6144

### CODE REVIEW

**Comment by taylq** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -62,6 +63,18 @@ def to_csv
         end
       end
     end
+
+    def from_omniauth auth
+      where(provider: auth.provider, uid: auth.uid).first_or_create do |user|
+        user.name = auth.info.name
+        user.email = auth.info.email
+        user.password = "123456"
```
Comment (huuphuc93) [inline app/models/user.rb]: cho ramdom password

**Comment by huuphuc93** [comment]
commented

**Comment by taylq** [comment]
ready

**Pull request merged**